### DISCUSSION TITLE
问一下 蓝奏云怎么用

### DISCUSSION THREAD

**Discussion by iowong** [AlistGo/alist#2411] [Q&A]
就是挂载分享的链接 不是挂载自己的 需要填写那几个 怎么填  求个例子  试了好几次都不行

**Comment by sevensky** [comment]
[看代码 drivers/lanzou/driver.go](https://github.com/alist-org/alist/blob/main/drivers/lanzou/driver.go)
链接类型不好用， 直接用cookie类型。 填cookie 保存就完事了

**Reply by aogg** [reply]
> [看代码 drivers/lanzou/driver.go](https://github.com/alist-org/alist/blob/main/drivers/lanzou/driver.go) 链接类型不好用， 直接用cookie类型。 填cookie 保存就完事了

报错

**Reply by aogg** [reply]
failed get objs: failed to list objs: login not

**Comment by aogg** [comment]
要给整个cookie，而不是cookie里的权限key，太坑了，文档写下吧
